### PR TITLE
fix(hf): remove early optimization of hf scheduler that is no longer needed

### PR DESCRIPTION
**Pull request by Yangqing**

### CODE REVIEW

File: leptonai/photon/hf/hf_utils.py [hunk 1]
```diff
@@ -38,8 +38,14 @@ def img_param_to_img(param):
 
 
 def create_diffusion_pipeline(task, model, revision, torch_compile=False):
-    from diffusers import DiffusionPipeline, DPMSolverMultistepScheduler
-    import torch
+    try:
+        from diffusers import DiffusionPipeline
+        import torch
+    except Exception:
```
Comment (bddppq) [inline leptonai/photon/hf/hf_utils.py:44]: `except ImportError` ?

**Review by bddppq — APPROVED** [review]

File: leptonai/photon/hf/hf_utils.py (hunk 1 above)
Comment (Yangqing) [reply leptonai/photon/hf/hf_utils.py]: done
